homeassistant/components/google_assistant/trait.py

```diff
@@ -2386,6 +2386,22 @@ class SensorStateTrait(_Trait):
     name = TRAIT_SENSOR_STATE
     commands: list[str] = []
 
+    def _air_quality_description_for_aqi(self, aqi):
+        if aqi is None or str(aqi).isnumeric() is False:
+            return "unknown"
+        if int(aqi) <= 50:
+            return "healthy"
```

Comment:
You end up re calculating the int:ing here.

Reply:
will adjust!